go/vt/vtctl/grpcvtctldserver/schema.go

```diff
@@ -0,0 +1,39 @@
+package grpcvtctldserver
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"vitess.io/vitess/go/vt/concurrency"
+	"vitess.io/vitess/go/vt/log"
+	"vitess.io/vitess/go/vt/mysqlctl/tmutils"
+	tabletmanagerdatapb "vitess.io/vitess/go/vt/proto/tabletmanagerdata"
+	topodatapb "vitess.io/vitess/go/vt/proto/topodata"
+	"vitess.io/vitess/go/vt/proto/vtctldata"
+	"vitess.io/vitess/go/vt/topo/topoproto"
+)
+
+// helper method to asynchronously diff a schema
```

Comment:
i'm still not clear on why we need this file, seems like this should still be in the main server.go file